### DISCUSSION TITLE
Option to change frequency of backups

### DISCUSSION THREAD

**Discussion by theksepyro** [FossifyOrg/General-Discussion#679] [Ideas]
I really appreciate the ability to automatically export contacts, which has saved me on multiple occasions. That said, I think it would be nice to have the ability to change the frequency at which the backups are made. For example once per week/month, or even "only when a change is made". 

I feel like the way things are now, I end up with a folder full of the exact same file over and over just with a different filename.

Thanks for the consideration!

**Comment by Aga-C** [comment]
Please create a feature request for it.

**Comment by Nikon6302** [comment]
> I really appreciate the ability to automatically export contacts, which has saved me on multiple occasions. That said, I think it would be nice to have the ability to change the frequency at which the backups are made. For example once per week/month, or even "only when a change is made". 
> 
> I feel like the way things are now, I end up with a folder full of the exact same file over and over just with a different filename.
> 
> Thanks for the consideration!

Where you created this issue?

**Reply by theksepyro** [reply]
https://github.com/FossifyOrg/Contacts/issues/184
